<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>FeaturesWindow</class>
    <widget class="QDialog" name="FeaturesWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>257</width>
                <height>269</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
                <widget class="Line" name="line">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QCheckBox" name="checkBoxClipboardShortcut">
                    <property name="text">
                        <string>Clipboard EVE Praisal shortcut</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="Line" name="line_2">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QCheckBox" name="checkBoxWormholeTypeKeybind">
                    <property name="text">
                        <string>Wormhole type keybinds</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="Line" name="line_3">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QCheckBox" name="checkBoxReminderBookmarkWormhole">
                    <property name="text">
                        <string>Reminder to bookmark wormholes</string>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Minimum</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>22</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <widget class="QCheckBox" name="checkBoxPlaySound">
                                    <property name="text">
                                        <string>Play Sound</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="labelSndPath">
                                    <property name="text">
                                        <string>PATH</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout">
                                    <item>
                                        <widget class="QPushButton" name="pushButtonTestSnd">
                                            <property name="text">
                                                <string>Play</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="pushButtonBrowseSnd">
                                            <property name="text">
                                                <string>Browse</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <widget class="QCheckBox" name="checkBoxFlashText">
                                    <property name="text">
                                        <string>Flash Text</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="Line" name="line_4">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QCheckBox" name="checkBoxSaveRefreshToken">
                    <property name="text">
                        <string>Save CREST refresh token</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>FeaturesWindow</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>FeaturesWindow</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
